cibuildwheel/linux.py

```diff
@@ -71,7 +73,11 @@ def get_python_configurations(build_selector: BuildSelector) -> List[PythonConfi
     ]
 
     # skip builds as required
-    target_archs = [platform.machine()]
+    target_archs = architectures or [Architecture(platform.machine())]
+    # x86_64 machines can run i686 docker containers
+    if Architecture.i686 not in target_archs and Architecture.x86_64 in target_archs:
+        target_archs.append(Architecture.i686)
```

Comment:
Shouldn't this only be appended if this is not specified manually?

Reply:
Yes, the `Architecture.i686 not in target_archs` part of the condition checks that it wasn't specified manually. I found this way is more explicit and allows to specify both, or i686 (without x86), or x86 (which includes both) on their own.

Reply:
If I do `cibuildwheel .` then I get the native archs (x86_64 and i686) but if I specify manually `cibuildwheel --archs "x86_64" .`, I should only get the x86_64 build. So adding the i686 build should only happen if the user doesn't specify archs in their options.

Reply:
I think "native", which is both the default and allowed in the list would be better. `native` would expand to `i686` and `x86_64`. Then you can do native + something else, or you can specify just one of these and not get the other.